proc_dash/utility.py

```diff
@@ -99,6 +108,9 @@ def get_pipelines_overview(bagel: pd.DataFrame) -> pd.DataFrame:
         "-".join(tup)
         for tup in pipeline_complete_df.columns.to_flat_index()
     ]
+    pipeline_complete_df = pipeline_complete_df.reindex(
```

Comment:
Is this just a thing that you also changed? I don't understand what issue this corresponds to.

Reply:
Aha, I thought I made a note of this in the commit message but I think I neglected to do so 🙈.

This change was basically a quick fix to ensure that the order in which pipeline columns (e.g., "freesurfer-7.0.2", "fmriprep-11.0.2") appear in the displayed datatable is the same as the order they appear along the axis of any plotted graphs. Since when used as inputs to plotting those names are sorted automatically, I figured I'd just sort the columns alphabetically in the table as well.